common-lib/src/main/resources/logback.xml

```diff
@@ -37,9 +37,18 @@
     <logger name="application" level="INFO" />
     <logger name="request" level="INFO" />
 
-    <root level="INFO">
-        <appender-ref ref="LOGFILE"/>
-        <appender-ref ref="ASYNCSTDOUT"/>
-    </root>
+    <if condition='property("STAGE").contains("DEV")'>
+      <then>
+        <root level="INFO">
+          <appender-ref ref="LOGFILE"/>
+        </root>
+      </then>
+      <else>
+        <root level="INFO">
+          <appender-ref ref="LOGFILE"/>
+          <appender-ref ref="ASYNCSTDOUT"/>
+        </root>
+      </else>
+    </if>
```

Comment:
could this be moved outside the condition, to avoid repetition and so its clearer that we still always have a log file? something like (written in GH UI, so might not be valid)...
```suggestion
<root level="INFO">
    <appender-ref ref="LOGFILE"/>
    <if condition='!property("STAGE").contains("DEV")'>
      <then>
          <appender-ref ref="ASYNCSTDOUT"/>
      </then>
    </if>
 </root>
```

Reply:
Cheers, tried it out locally and it's working great so added as a commit

Reply:
probs need to check it on TEST just to make sure the negation actually works and it still logs to stdout

Reply:
deployed to TEST, logs are still coming through as expected there

Reply:
nice